src/Actions/AdminMenus.php

```diff
@@ -25,6 +25,14 @@ class AdminMenus {
 	 * @return void
 	 */
 	public function bcgov_block_theme_menu() {
+        add_menu_page(
+            __( 'Using BCGov Block Theme', 'bcgov-block-theme' ),
+            __( 'Using the BCGov Block Theme', 'bcgov-block-theme' ),
+            'edit_others_posts',
```

Comment:
I wonder if we can be even less strict than this because I think this would exclude users with the author role for example who would still benefit from being able to see the documentation. I feel like anyone who has access to wp-admin/ might as well have this menu item since the documentation itself is public.

Reply:
I wondered about that myself

Reply:
Yeah I think I shouldn't have specified editors in the ticket. Anyone should be able to see it.
@ShawnTurple What do you think about using a less restrictive capability?

Reply:
i think edit_own_posts if that is such a cap, should be used, and this would include authors

Reply:
`edit_posts` seems like a good one. Includes all roles above subscriber.